docs/multiple-stolon.md

```diff
@@ -0,0 +1,50 @@
+# Installing multiple stolon clusters in one Gravity cluster
+This document provides an instruction on how to build and install multiple stolon clusters 
+in one Gravity cluster.
+
+## Requirements
+
+- tele binary: 5.5+
+- stolon application: 1.13.0+
+
+## Building
+By default using provided [Makefile](../Makefile) stolon application is built as a Gravity application and could be installed in Gravity cluster with `gravity install` from resulting tarball.  
+For the additional stolon(s) in cluster [Application Catalog](https://gravitational.com/gravity/docs/catalog/) feature could be reused. Application does not need to be pushed to catalog, but the process of making of that application image type applies here.
```

Comment:
`To add (or introduce) additional stolon instances into the cluster, use the [Application Catalog]() feature.`

Not sure I understood the following sentence though - do you mean to say that the same approach as implemented in the Makefile equally applies here?

Reply:
I mean our Application catalog instructions contain pushing an app to a Hub, but, technically, it is not needed. We can just create tarball and use it with `gravity app install app-version.tar`.